povm_toolbox/post_processor/median_of_means.py

```diff
@@ -90,6 +91,9 @@ def __init__(
                 :meth:`get_decomposition_weights`. When this is ``None``, the default
                 "state-average" Dual frame will be constructed from the POVM stored in the
                 ``povm_sample``'s :attr:`.POVMPubResult.metadata`.
+            combine_counts: indicates, when applicable, whether to combine the counts associated
+                with different parameter values that were submitted for a single parametrized
+                circuit.
```

Comment:
This needs to be aligned with whatever updates we apply based on the previous comment.